tests/dummy/app/styles/app.css

```diff
@@ -42,11 +42,7 @@
     border-color: #FFC107;
 }
 
-.popper[x-placement^="top"] {
-    margin-bottom: 5px;
```

Comment:
The padding between the popper and its reference element is now managed with `offset` modifier https://popper.js.org/docs/v2/migration-guide/#4-remove-all-css-margins